### DISCUSSION TITLE
필터 주소가 바뀐것 같은데

### DISCUSSION THREAD

**Discussion by Maccvc** [List-KR/community#29] [Q&A]
AdGuard for iOS에서 !#include나 !#if 같은 전처리 지시자가 아직 지원되지 않기에 
List-KR DNS 필터 설치가 아닌 경우 기본적으로 설치되어 있는 List-KR 필터를 사용해주세요.

macOS와 iOS에서 Adguard 패밀리 플랜을 사용중입니다
위 문구대로 
DNS 필터는 바뀐 주소(https://cdn.jsdelivr.net/gh/adguardteam/HostlistsRegistry@main/assets/filter_25.txt) 를 사용하고
사파리는 adguard에 포함된 list-kr 필터를 쓰면 되는게 맞나요

추후에 adguard가 전처리 지시자(?)를 지원하도록 업데이트가 되면 
adguard앱에 포함된 list-kr 필터도 바뀐 주소로 업데이트가 되는것인가요?

지금 adguard에 기본으로 포함된 list-kr 필터를 안쓰고 
https://cdn.jsdelivr.net/gh/List-KR/List-KR@master/filter-AdGuard.txt
위의 필터를 추가해서 쓰는데 
제 사용 범위에선 딱히 이상은 없는거 같긴 하네요 ㅎㅎ

**Comment by piquark6046** [comment]
> 사파리는 adguard에 포함된 list-kr 필터를 쓰면 되는게 맞나요

AdGuard for Safari에서 수동으로 설치된 List-KR의 규칙 수가 0이 아니면 사용이 가능합니다.

> 추후에 adguard가 전처리 지시자(?)를 지원하도록 업데이트가 되면
adguard앱에 포함된 list-kr 필터도 바뀐 주소로 업데이트가 되는것인가요?

약간 오해가 있으신 것 같네요. AdGuard에 기본적으로 포함되어 있는 List-KR은 AdGuard의 [FiltersCompiler](https://github.com/AdguardTeam/FiltersCompiler)라고 설치되는 확장 프로그램과 독립 프로그램는 다르게 AdGuard나 GitHub 서버에서 전처리 지시자를 읽어서 각 플렛폼에 맞게 컴파일하여 한 파일로 통합하게끔 처리합니다.
이러한 한 개의 파일로 처리된 List-KR 필터는 수동으로 설치하시는 List-KR 필터와 크게 다르지 않습니다.
또한 이러한 필터는 AdGuard의 자체 서버인 filters.adtidy.org 을 통해 배포되며 필터에 대한 구독 URL 또한 변경되지 않습니다.

일반적으로 4시간 보다 짧은 기간으로 필터 업데이트하시는 분에 한해 수동으로 설치하시는 방향으로 선택합니다.
더 긴 주기로 필터 업데이트를 하시면 기본적으로 포함되어 있는 List-KR을 사용하셔도 상관 없습니다.

> 위 문구대로
DNS 필터는 바뀐 주소(https://cdn.jsdelivr.net/gh/adguardteam/HostlistsRegistry@main/assets/filter_25.txt) 를 사용하고

기본 GitHub RAW 파일에서 jsDelivr 으로 변경한 이유는 더 빠른 로드, 데이터 전송 중에 발생하는 탄소를 줄이기 위함 그리고 GitHub 서버의 트래픽 감소가 목적입니다.
아무튼, DNS 차단 리스트 목적으로 사용하신다면 올바른 URL을 사용하시고 계십니다.

**Reply by Maccvc** [reply]
감사합니다!
